src/devices/Bmxx80/Bme680.cs

```diff
@@ -435,24 +430,7 @@ public override bool TryReadTemperature(out Temperature temperature)
         /// the measurement was valid. Undefined otherwise.
         /// </param>
         /// <returns><code>true</code> if measurement was not skipped, otherwise <code>false</code>.</returns>
-        public bool TryReadGasResistance(out ElectricResistance gasResistance)
-        {
-            if (!ReadGasMeasurementIsValid() || !ReadHeaterIsStable())
-            {
-                gasResistance = default;
-                return false;
-            }
-
-            // Read 10 bit gas resistance value from registers
-            var gasResRaw = Read8BitsFromRegister((byte)Bme680Register.GAS_RES);
-            var gasRange = Read8BitsFromRegister((byte)Bme680Register.GAS_RANGE);
-
-            var gasRes = (ushort)((ushort)(gasResRaw << 2) + (byte)(gasRange >> 6));
-            gasRange &= (byte)Bme680Mask.GAS_RANGE;
-
-            gasResistance = CalculateGasResistance(gasRes, gasRange);
-            return true;
-        }
+        public bool TryReadGasResistance(out ElectricResistance gasResistance) => TryReadGasResistanceCore(out gasResistance);
```

Comment:
What do you gain from having Core meethod? wouldn't it be better to directly call TryReadGasResistance?